data/src/main/java/com/github/leuludyha/data/repository/datasourceImpl/BitmapProviderImpl.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.leuludyha.data.repository.datasourceImpl
+
+import android.graphics.Bitmap
+import com.github.leuludyha.data.repository.datasource.BitmapProvider
+import com.squareup.picasso.Picasso
+
+/**
+ * Fetches a [Bitmap] online.
+ */
+class BitmapProviderImpl: BitmapProvider {
+    override fun invoke(path: String): Bitmap? {
+        return try {
+            Picasso.get().load(path).get()
+        } catch(e: Exception) {
+            null
```

Comment:
Maybe it would be smart to add a default image of this type such as this one to indicate that the image couldn't be fetched instead of dealing with it on a case by case basis in each instance.
 <img src="https://media.istockphoto.com/id/1336657186/vector/no-wi-fi-flat-vector.jpg?s=612x612&w=0&k=20&c=HbcdNJXVwQl3UhnENheZy0VXLXVrPDebCWD9aBHVDJM=" width="100" height="100">

Reply:
Added to next sprint.